pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -508,6 +508,9 @@ func (m *cgroupManagerImpl) Pids(name CgroupName) []int {
 			// do nothing, continue
 			continue
 		}
+		if err != nil {
+			continue
```

Comment:
Should we always be silently ignore this failure? It makes sense to me that we do that for the `os.IsNotExist` error above, as it means the pod cgroup has already been deleted, but what if the error is because of permission errors or another type of issue?

Perhaps we could add a helpful log message?

Reply:
I looked at the source code for getCgroupProcs which is called next.
I think if there is error here, getCgroupProcs would encounter error as well (which is checked).